src/Mod/AddonManager/AddonManagerTest/app/test_utilities.py

```diff
@@ -132,21 +131,22 @@ def test_get_assigned_string_literal(self):
             result = get_assigned_string_literal(line)
             self.assertIsNone(result)
 
-    def test_get_macro_version_from_file(self):
-        if FreeCAD:
-            test_dir = os.path.join(
-                FreeCAD.getHomePath(), "Mod", "AddonManager", "AddonManagerTest", "data"
-            )
-            good_file = os.path.join(test_dir, "good_macro_metadata.FCStd")
```

Comment:
If this test file is not needed any longer, it should be removed from the source tree (dito for bad_macro_metadata.FCStd below)

Reply:
Thanks for the reminder, I've removed them now.